<launch>
    <arg name="joy_dev" default="/dev/input/js0" />
    <arg name="joy_topic" default="joy" />

    <node pkg="drone_teleop" type="drone_ardupilot_teleop" name="drone_ardupilot_teleop" clear_params="true" output="screen">
        <remap from="joy" to="joy"/>
        <remap from="drone_twist" to="drone_twist"/>
        <rosparam file="$(find drone_teleop)/config/param.yaml"/>
    </node>

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" value="$(arg joy_dev)" />
        <param name="deadzone" value="0.3" />
        <param name="autorepeat_rate" value="20" />
        <remap from="joy" to="$(arg joy_topic)" />
    </node>
</launch>